Add example UI (#8)
OffchainLabs/stylus-by-example: src/app/applications/vending_machine/page.mdx

export const metadata = {
  title: "Vending Machine • Stylus by Example",
  description:
    "An example implementation of the Vending Machine in Rust using Arbitrum Stylus.",
};

{/* Begin Content */}

# Vending Machine

An example project for writing Arbitrum Stylus programs in Rust using the [stylus-sdk](https://github.com/OffchainLabs/stylus-sdk-rs). It includes a Rust implementation of a vending machine Ethereum smart contract.

- distribute Cupcakes to any given address
- count Cupcakes balance of any given address

Here is the interface for Vending Machine.

```solidity
interface IVendingMachine {
    // Function to distribute a cupcake to a user
    function giveCupcakeTo(address userAddress) external returns (bool);

    // Getter function for the cupcake balance of a user
    function getCupcakeBalanceFor(address userAddress) external view returns (uint);
}
```

> Example implementation of a MultiSig Wallet contract written in Rust.

Reviewer: ```suggestion
Example implementation of the Vending Machine contract written in Rust.
```
